dev-infrastructure/templates/automation-account.bicep

```diff
@@ -11,10 +14,10 @@ param dailyScheduleStartTime string = '${substring(dateTimeAdd(utcNow(), 'P1D'),
 param ntlyScheduleStartTime string = '${substring(dateTimeAdd(utcNow(), 'P1D'), 0, 10)}T00:00:00Z'
 
 module automationAccount '../modules/automation-account/account.bicep' = {
-  name: 'hcp-dev-automation'
+  name: format('hcp-{0}-automation', environment)
```

Comment:
nit: we can use string interpolation available in current bicep, e.g. 'hcp-${environment}-automation'